packages/kbn-ecs-schema/src/schemas/index.ts

```diff
@@ -0,0 +1,203 @@
+      'Example: `docker` and `k8s` labels.',
+    example: '{"application": "foo-bar", "env": "production"}',
+    flat_name: 'labels',
+    level: 'core',
+    name: 'labels',
+    normalize: [],
+    object_type: 'keyword',
+    short: 'Custom key/value pairs.',
+    type: 'object'
+  },
+  message: {
+    dashed_name: 'message',
+    description: 'For log events the message field contains the log message, optimized for viewing in a log viewer.\n' +
+      'For structured logs without an original message field, other fields can be concatenated to form a human-readable summary of the event.\n' +
+      'If multiple messages exist, they can be combined into one message.',
+    example: 'Hello World',
+    flat_name: 'message',
+    level: 'core',
+    name: 'message',
+    normalize: [],
+    short: 'Log message optimized for viewing in a log viewer.',
+    type: 'match_only_text'
+  },
+  tags: {
+    dashed_name: 'tags',
+    description: 'List of keywords used to tag each event.',
+    example: '["production", "env2"]',
+    flat_name: 'tags',
+    ignore_above: 1024,
+    level: 'core',
+    name: 'tags',
+    normalize: [ 'array' ],
+    short: 'List of keywords used to tag each event.',
+    type: 'keyword'
+  },
+  span: {
+    id: {
+      dashed_name: 'span-id',
+      description: 'Unique identifier of the span within the scope of its trace.\n' +
+        'A span represents an operation within a transaction, such as a request to another service, or a database query.',
+      example: '3ff9a8981b7ccd5a',
+      flat_name: 'span.id',
+      ignore_above: 1024,
+      level: 'extended',
+      name: 'span.id',
+      normalize: [],
+      short: 'Unique identifier of the span within the scope of its trace.',
+      type: 'keyword'
+    }
+  },
+  trace: {
+    id: {
+      dashed_name: 'trace-id',
+      description: 'Unique identifier of the trace.\n' +
+        'A trace groups multiple events like transactions that belong together. For example, a user request handled by multiple inter-connected services.',
+      example: '4bf92f3577b34da6a3ce929d0e0e4736',
+      flat_name: 'trace.id',
+      ignore_above: 1024,
+      level: 'extended',
+      name: 'trace.id',
+      normalize: [],
+      short: 'Unique identifier of the trace.',
+      type: 'keyword'
+    }
+  },
+  transaction: {
+    id: {
+      dashed_name: 'transaction-id',
+      description: 'Unique identifier of the transaction within the scope of its trace.\n' +
+        'A transaction is the highest level of work measured within a service, such as a request to a server.',
+      example: '00f067aa0ba902b7',
+      flat_name: 'transaction.id',
+      ignore_above: 1024,
+      level: 'extended',
+      name: 'transaction.id',
+      normalize: [],
+      short: 'Unique identifier of the transaction within the scope of its trace.',
+      type: 'keyword'
+    }
+  },
+  agentEcs: {...agentEcs},
```

Comment:
Can we change these names from `<fieldset>Ecs` to `<fieldset>`?

I believe it'll make for cleaner naming when referencing fields: `ecsSchema.service.address` vs. `ecsSchema.serviceEcs.address`.

Reply:
The best I can do is updating to `agent: {...agentEcs}`

Due to 'package' and 'interface' being 'reserved' words, we have to be a little more creative.